assets/sass/components/global/_googlesitekit-entity-search.scss

```diff
@@ -42,7 +42,17 @@
 		.googlesitekit-entity-search__close {
 			height: 32px;
 			min-width: 36px;
-			padding: 0;
+
+			/**
+			*	We need to set 5px left and right padding for
+			*	larger screens since we have set the close button
+			*	icon size 20 for consistency.
+			*/
+			padding: 0 5px;
+
+			@media (max-width: $bp-mobileOnly ) {
+				padding: 0;
+			}
```

Comment:
If we don’t want these changes, we need to revert the `CloseDark` icon to have the width `30`.

```javascript
<CloseDark width="20" height="20" />
```

Reply:
@hussain-t Let's set it to 30 but see my previous comment about its dimensions.